tests/helpers.py

```
@@ -119,8 +119,21 @@ def runs(
         return runs
 
 
+class FakeUser:
+    def __init__(self, *, user_id: str, username: str) -> None:
+        self.id = user_id
+        self.username = username
+        self._secret = "hidden"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "_secret"

(S105)